.changeset/nasty-dolls-burn.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/lexical-library': minor
+'@twilio-paste/core': minor
+---
+
+[Lexical Library] add export for: CLEAR_EDITOR_COMMAND, CLEAR_EDITOR_COMMAND, KEY_ENTER_COMMAND, COMMAND_PRIORITY_CRITICAL, COMMAND_PRIORITY_HIGH, COMMAND_PRIORITY_NORMAL, COMMAND_PRIORITY_LOW, and COMMAND_PRIORITY_EDITOR
```

Comment:
nitpick: it's usually more readable to put content like this into a bulleted list (not a big thing tho, so for next time!)